odl/solvers/nonsmooth/proximal_operators.py

```diff
@@ -146,7 +146,7 @@ def convex_conj_prox_factory(sigma):
         proximal : `Operator`
             The proximal operator of ``s * F^*`` where ``s`` is the step size
         """
-        sigma = float(sigma)
+        sigma = float(abs(sigma))
```

Comment:
Are we sure that this is the correct formula in general? I'm frankly not sure but we could use input from @aringh and Sebastian who know some optimization theory.

Also note the issue  #590 where we discuss this in depth.

Reply:
I'm not entirely sure on that either and am not an expert on the mathematics of optimization (I am a biomedical engineer by training with expertise primarily in MRI acquisition methods and medical image reconstruction).  It seems to work okay for the denoising example provided here, but I agree that we should wait for expert feedback on the correct approach.